document-service/documents-ui/src/app.config.ts

```diff
@@ -0,0 +1,192 @@
+export default defineAppConfig({
+  ui: {
```

Comment:
the extension will also pickup all of this as well fyi. I would remove and add a comment with the link to the app.config in the business-dashboard-ui so that its clear its taking that one and then applying things in here on top of it (default nuxt/ui -> extended layer app.configs -> this app.config)

Reply:
Ok great, will do!

Reply:
Updated with comments